src/libraries/System.Private.CoreLib/src/System/Runtime/Intrinsics/Vector128.cs

```diff
@@ -2419,6 +2435,32 @@ public static Vector128<sbyte> Shuffle(Vector128<sbyte> vector, Vector128<sbyte>
             return result;
         }
 
+        /// <summary>Creates a new vector by selecting values from an input vector using a set of indices.
+        /// Behavior is platform-dependent for out-of-range indices.</summary>
```

Comment:
Right. For the xplat hardware intrinsics we strive to expose deterministic functionality as otherwise it easily leads to bugs and other subtleties between platforms.

We expose `Unsafe` variants to allow users to"opt-in" to non-deterministic functionality, such as leaving upper-bits undefined or doing platform specific things.

In this case, Arm64 will treat any "out of bounds" access as "select the value 0", where-as x86/x64 will only do that if the most-significant bit of the index is set. For values where the most-significant bit is not set, it masks the index to be between 0 and Count.

Reply:
Thanks. Can we include those details in Remarks on the method?

Reply:
Something like this?

> Unlike Shuffle, this method does not ensure that values for out-of-range indices are set to zero. Behavior for such indices is platform-dependent.

It feels like it's just repeating the description again, is there something else you had in mind that should be emphasized?

Reply:
"In this case, Arm64 will treat any "out of bounds" access as "select the value 0", where-as x86/x64 will only do that if the most-significant bit of the index is set. For values where the most-significant bit is not set, it masks the index to be between 0 and Count."